### PR TITLE
Stockprices api connected

### PR DESCRIPTION
**Pull request by beckskelly**
Pulled in data from month api and year. No date bar at the moment until we can fix it

### CODE REVIEW

File: src/components/stockVisualisationComponents/ChartTypes/PriceChart/PriceChart.js [hunk 1]
```diff
@@ -10,45 +10,85 @@ import {
...
-        { name: '2017', price: 400 },
-        { name: '2018', price: 10 },
-        { name: '2019', price: 60 },
-        { name: '2020', price: 700 },
-        { name: '2021', price: 100 }]
+        { date: '2022-01-10', price: 400 },
+        { date: '2022-01-11', price: 700 },
+        { date: '2022-01-12', price: 60 },
+        { date: '2022-01-13', price: 700 },
+        { date: '2022-01-1', price: 500 }]
```
Comment (BearachB) [inline src/components/stockVisualisationComponents/ChartTypes/PriceChart/PriceChart.js:75]: Should these be hardcoded?

**Review by BearachB — COMMENTED** [review]
Are the day values supposed to be hardcoded in?

**Review by joeycor92 — CHANGES_REQUESTED** [review]
Getting a few errors which should be easy to fix.
image

The charts really do look fantastic and the transitions are brilliant when going from period to another. Unreal

File: src/components/stockVisualisationComponents/ChartTypes/PriceChart/PriceChart.js (hunk 1 above)
Comment (joeycor92) [reply src/components/stockVisualisationComponents/ChartTypes/PriceChart/PriceChart.js:75]: They should be, we don't have daily data yet. My weekend job.

**Comment by beckskelly** [comment]
@joeycor92 sorted those errors now! get to da chopper

File: src/components/stockVisualisationComponents/ChartTypes/PriceChart/PriceChart.js (hunk 1 above)
Comment (BearachB) [reply src/components/stockVisualisationComponents/ChartTypes/PriceChart/PriceChart.js:61]: Grand so! Wasn't positive

**Review by joeycor92 — APPROVED** [review]
Looks good now. Thanks for getting rid of those errors.

**Comment by BearachB** [comment]
Looks really good, well done @beckskelly!

**Pull request merged**